install_config/persistent_storage/persistent_storage_flex_volume.adoc

```diff
@@ -0,0 +1,304 @@
+[[install-config-persistent-storage-persistent-storage-flex-volume]]
+= Persistent Storage Using Flex Volume Plugins
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+While {product-title} has built-in many
+xref:index.adoc#[volume plugins]
+to use various storage technologies, sometimes it may be necessary to consume
+storage from a backend that does not have a plugin in {product-title}.
+{product-title} can be extended via Flex Volume drivers to access such storage
+backend and provide
+xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent storage]
+to applications.
+
+Flex Volume driver is a binary, typically a shell script, deployed to all
+machines in the cluster, both masters and nodes as part of installation.
+
+[NOTE]
+====
+Flex Volume is an alpha feature and may change in a future release of {product-title}.
+====
+
+=== Installation
```

Comment:
[[flex-volume-installation]]
== Installation

(each heading needs a unique ID and we can keep the Installation heading and following headings in line with Overview)